src/OpenConext/Profile/Value/Consent.php

```diff
@@ -95,4 +113,12 @@ public function getConsentType()
     {
         return $this->consentType;
     }
+
+    /**
+     * @return null|AttributeSet
+     */
+    public function getAttributes()
+    {
+        return $this->attributes;
```

Comment:
Disagree, it is a value object. Therefor, it has a known value, so either it should be split out to a new attribute  (eeehr, something like ConsentWithAttributes), or the consumer should be responsible for ensuring compatibility with the known api: `null|AttributeSet`. Has-er may be added for convenience, but certainly not a LogicException - it is documented that a Consent can exist with or without Attrributes, so assuming attributes exists at all times conflicts with documented behavior.

A different object has my preference btw :tongue: